### PR TITLE
feat: add Opitx

### PR DESCRIPTION
**Pull request by Thomashighbaugh**

### CODE REVIEW

**Comment by tmdh** [comment]
Rename `optix.yml` to `opitx-md-editor.yml` and rename `logo.png` to `opitx-md-editor-icon.png`

File: apps/opitx-md-editor/opitx.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: Opitx Markdown Editor
+description: 'A markdown editor for those tired of loosing their work'
```
Comment (ckerr) [inline apps/opitx-md-editor/opitx.yml]:
```suggestion
description: 'A markdown editor for those tired of losing their work'
```

File: apps/opitx-md-editor/opitx.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+website: 'https://opitx-docs.netlify.com'
+repository: 'https://github.com/Thomashighbaugh/Opitx'
+keywords:
+    - React
+    - Markdown
+    - Note
+    - Electron
```
Comment (ckerr) [inline apps/opitx-md-editor/opitx.yml]: Everything in the repo is an Electron app -- this keyword probably isn't necessary :slightly_smiling_face:

**Comment by HashimotoYT** [comment]
Closing due to inactivity. Please open new PR if to add your application.

**Pull request closed without merging**